index.html

```diff
@@ -11,65 +11,79 @@
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
-
-        <section class="bottom-content">
 
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-
-        </section>
-        
-        <footer>
-            <nav>
-                <a href="#">Home</a>
-                <a href="#">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
-        </footer>
-    </div><!-- container -->        
-</body>
+<body>
+  
+    <header>
+    <nav>
+        <img src="img/lambda-black.png" alt="">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="/">Products</a>
+        <a href="/">Blog</a>
+        <a href="/">Contact</a>
+      </nav>
+      <img src="img/jumbo.jpg" alt="" class="header-img">
+    </header>
+      <div class="container">
+     
+          <section class="top-content">
+              <div class="text-container">
+                  <h2>The Future</h2>
+                  <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+              </div>
+              <div class="text-container">
+                  <h2>The Past</h2>
+                  <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+              </div>
+          </section>
+          
+          <section class="middle-content">
+          
+              <h2>Why Did It Have To Be Boxes...</h2>
+              
+              <div class="boxes">
+                  <div class="box" id="one">Box 1</div>
+                  <div class="box" id="two">Box 2</div>
```

Comment:
Awesome, you utilized the id attribute to handle each color change on the boxes. You can also add additional class names to the element if you want to avoid using the id attribute. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```